src/actions/smartWalletActions.js

```diff
@@ -1793,38 +1801,45 @@ export const migrateEnsFromArchanovaToEtherspotAction = (
       await dispatch(switchAccountAction(getAccountId(archanovaAccount)));
     }
 
-    const accountAssets = accountAssetsSelector(getState());
-    const supportedAssets = supportedAssetsSelector(getState());
-    const ethAsset = getAssetData(getAssetsAsList(accountAssets), supportedAssets, ETH);
-
-    const completeTransactionPayload = migratorTransactions
-      .map(({ data, to }) => ({
-        to,
-        data,
-        amount: 0,
-        symbol: ethAsset.symbol,
-        decimals: ethAsset.decimals,
-        contractAddress: ethAsset.address,
-        txFeeInWei: feeInfo?.fee,
-        gasToken: feeInfo?.gasToken,
-      }))
-      .reduce((transactionPayload, transaction, index) => {
-        if (index === 0) {
-          return {
-            ...transaction,
-            sequentialTransactions: [],
-            extra: { isENSMigrationToEtherspot: true },
-          };
-        }
+    let hash;
+    try {
+      hash = await archanovaService.sendRawTransactions(rawTransactions);
+    } catch (error) {
+      reportErrorLog('migrateEnsFromArchanovaToEtherspotAction failed', {
+        error,
+        archanovaAccount,
+        activeAccount,
+        rawTransactions,
+      });
+    }
```

Comment:
Are these methods throwing raw errors, is this why we're try catching?

Reply:
Yes. I guess these questions arise of me trying to adopt a pattern of await without `catch`,  but use `try/catch` instead. Will see how it goes in long term, this is per @mdjastrzebski inisghts.